spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -236,17 +272,31 @@ def then_i_see_voided_declarations
 
   def then_i_should_see_correct_service_fee_payment_breakdown
     within first(".app-application__card") do
-      expect(page).to have_css("tr:nth-child(2) td:nth-child(1)", text: "Service fee")
-      expect(page).to have_css("tr:nth-child(2) td:nth-child(2)", text: npq_leading_behaviour_culture_contract.recruitment_target)
-      expect(page).to have_css("tr:nth-child(2) td:nth-child(3)", text: number_to_pounds(17.05))
-      expect(page).to have_css("tr:nth-child(2) td:nth-child(4)", text: number_to_pounds(1_227.79))
+      row = (show_targeted_delivery_funding? ? 3 : 2)
+      expect(page).to have_css("tr:nth-child(#{row}) td:nth-child(1)", text: "Service fee")
+      expect(page).to have_css("tr:nth-child(#{row}) td:nth-child(2)", text: npq_leading_behaviour_culture_contract.recruitment_target)
+      expect(page).to have_css("tr:nth-child(#{row}) td:nth-child(3)", text: number_to_pounds(17.05))
+      expect(page).to have_css("tr:nth-child(#{row}) td:nth-child(4)", text: number_to_pounds(1_227.79))
+    end
+  end
+
+  def then_i_should_see_correct_targeted_delivery_funding_breakdown
+    within first(".app-application__card") do
+      expect(page).to have_css("tr:nth-child(2) td:nth-child(1)", text: "Targeted delivery funding")
+      expect(page).to have_css("tr:nth-child(2) td:nth-child(2)", text: 1)
+      expect(page).to have_css("tr:nth-child(2) td:nth-child(3)", text: number_to_pounds(100.0))
+      expect(page).to have_css("tr:nth-child(2) td:nth-child(4)", text: number_to_pounds(100.0))
     end
   end
 
   def then_i_should_see_the_correct_total
     within first(".app-application__card") do
       expect(page).to have_content("Course total")
-      expect(page).to have_content(number_to_pounds(1_458 + 1_227.79))
+      if show_targeted_delivery_funding?
```

Comment:
is it possible to create a new expectation to avoid including logic for targeted delivery funding cohort?  Same for the rest of the file